packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts

```diff
@@ -25,7 +25,7 @@ export class CloudinaryMediaStore implements MediaStore {
   fetchFunction = (input: RequestInfo, init?: RequestInit) => {
     return fetch(input, init)
   }
-  accept = '*'
+  accept = 'text/*,  application/*, image/*'
```

Comment:
what does it mean concretely for Cloudinary users, what type of files are now supported?

Reply:
These are MIME types:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

I couldn't find a list of _just_ `text`, `application`, or `image` types, but if you go through that list, you'll see the `MIME Type` column.  If the `MIME Type` begins with `text/`, `application/` or `image/`, we'll accept it.

I'm not quite sure there is an exhaustive list (especially for `application/` where that could be any application's file type).

Reply:
So we accept things like zip files now, should we be concerned about security?

Reply:
Could we get support for `audio/` MIME type?